client/src/app/util/EmailVerification.js

```diff
@@ -0,0 +1,21 @@
+import emailjs, {init} from '@emailjs/browser';
+init('5A9OSqglcYcpKAMgd');
```

Comment:
what is this init for? do we need '5A9OSqglcYcpKAMgd' to be private?

Reply:
That is the emailjs API key. I think its fine being public, emailjs gives a public api key (the one I used in the code) as well as a private api key, so we should be good.